nemo/collections/asr/data/audio_to_text_lhotse_prompted.py

```diff
@@ -180,8 +199,9 @@ def canary(
         )
         prompts_with_answers.append(encoded["input_ids"])
         prompts.append(encoded["context_ids"])
+        answers.append(encoded["answer_ids"][:-1])  # Strip Canary's EOS
```

Comment:
Since the current issue comes from stripping buried int the code, let's have a bool here so we're aware that this is happening.